packages/twenty-front/src/modules/object-record/record-index/export/hooks/__tests__/useExportFetchRecords.test.ts

```diff
@@ -7,9 +7,9 @@ import {
 } from '../useExportFetchRecords';
```

Comment:
Note: we have hooks with same name: useExportFetchRecords in different folders. Looks duplicated code?